modules/imgproc/src/filter.simd.hpp

```diff
@@ -465,6 +465,87 @@ struct RowVec_8u32s
     bool smallValues;
 };
 
+struct RowVec_8u32f
+{
+    RowVec_8u32f() { smallValues = false; }
+    RowVec_8u32f( const Mat& _kernel )
+    {
+        kernel = _kernel;
+        smallValues = true;
+        int k, ksize = kernel.rows + kernel.cols - 1;
+        for( k = 0; k < ksize; k++ )
+        {
+            int v = kernel.ptr<int>()[k];
+            if( v < SHRT_MIN || v > SHRT_MAX )
+            {
+                smallValues = false;
+                break;
+            }
+        }
```

Comment:
Is this needed? You don't use the `smallValues` anywhere in the implementation of `operator()`

Reply:
Oh, I think you have to leave the lines that assigned `_kernel` to `kernel`, because you read the value of `kernel` in `operator()`.